demo/src/app/components/datepicker/overview/datepicker-overview.component.ts

```diff
@@ -73,6 +73,13 @@ providers: [{provide: NgbDateParserFormatter, useClass: YourOwnParserFormatter}]
 </ng-template>
 
 <ngbDatepicker [dayTemplate]=“t”/>
+`,
+    footerTemplate: `
+<ng-template #t let-datepicker>
+  <button (click)="datepicker.onDateSelect(today)">Today</button>
```

Comment:
again should be `model = today` or `d.close()` as and example, not `onDateSelect()`